changelogs/unreleased/gh-6292-fix-dropping-messages-in-case-when-connection-closing.md

```diff
@@ -0,0 +1,4 @@
+## bugfix/core
```

Comment:
In the commit message please add an empty line before `Closes`.

Reply:
fixed